### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by jkongie**
Updates the code owners file to provide more specific ownership for packages

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1,4 +1,33 @@
...
+/core/services/fluxmonitorv2 @jkongie @PiotrTrzpil @spooktheducks @connorwstein
+/core/services/gasupdater @samsondav @PiotrTrzpil
+/core/services/headtracker @PiotrTrzpil
+/core/services/health @archseer
+/core/services/job @spooktheducks @connorwstein
+/core/services/keeper @RyanRHall
+/core/services/log @PiotrTrzpil @spooktheducks
+/core/services/offchainreporting @spooktheducks @connorwstein
+/core/services/periodicbackup @PiotrTrzpil
+/core/services/pipeline @spooktheducks @connorwstein
```
Comment (archseer) [inline CODEOWNERS:15]: Can you list me under the pipeline as well?

Comment (jkongie) [reply CODEOWNERS]: Done

**Review by archseer — DISMISSED** [review]

File: CODEOWNERS [hunk 2]
```diff
@@ -1,4 +1,33 @@
...
+/core/cmd @jkongie
+
+# UI
+/operator-ui/ @DeividasK @jkongie
+
+# Contracts
 /evm-contracts/ @alexroan @se3000
-/contracts/ @alexroan @se3000
-/operator-ui/ @DeividasK
+/contracts/ @alexroan @se3000
```
Comment (connorwstein) [inline CODEOWNERS:33]: can I haz perms on this one too - currently adding VRFCoordinatorV2 contracts in here

File: CODEOWNERS [hunk 3]
```diff
@@ -1,4 +1,33 @@
...
+/core/services/gasupdater @samsondav @PiotrTrzpil
+/core/services/headtracker @PiotrTrzpil @samsondav
+/core/services/health @archseer @samsondav
+/core/services/job @spooktheducks @connorwstein @archseer
+/core/services/keeper @RyanRHall @samsondav
+/core/services/log @PiotrTrzpil @spooktheducks
+/core/services/offchainreporting @spooktheducks @connorwstein @samsondav
+/core/services/periodicbackup @PiotrTrzpil @samsondav
+/core/services/pipeline @spooktheducks @connorwstein @archseer
+/core/services/postgres
```
Comment (connorwstein) [inline CODEOWNERS:16]: is this the same behaviour as not mentioning the package here? And would it imply that everyone is an owner of it or no one is?

Comment (jkongie) [reply CODEOWNERS:16]: I would say that this implies there are no owners. 

 I've left it in for us to fill out as we need to but no one has claimed those as of yet.

**Review by archseer — APPROVED** [review]
Let's merge this as is, we can figure out the remaining three folders afterwards.

**Pull request merged**